docs/user/README.md

```diff
@@ -106,13 +106,14 @@ to load path and require it in `init.el`
   - Note that if you use a plugin manager other than `vim-plug`, you may need to manually
     checkout the `release` branch wherever your plugin manager cloned it. Otherwise you will
     get errors about a missing javascript file.
-* Run `:CocInstall coc-rust-analyzer` to install `coc-rust-analyzer`, this extension already implemented _almost_ all features supported in VSCode extension:
+* Run `:CocInstall coc-rust-analyzer` to install [coc-rust-analyzer], this extension implemented _most_ of the features supported in the VSCode extension:
   - same configurations as VSCode extension, `rust-analyzer.raLspServerPath`, `rust-analyzer.enableCargoWatchOnStartup` etc.
   - same commands too, `rust-analyzer.analyzerStatus`, `rust-analyzer.startCargoWatch` etc.
-  - highlighting and inlay_hints is not works by now
+  - highlighting and inlay_hints is not implemented yet
```

Comment:
Are not?

Reply:
Also, aren't both highlighting and inlay hints implemented already?
That's not the final implementation for sure, but far from "not implemented" state already :)

Reply:
@SomeoneToIgnore VSCode-RA uses Decorations to make highlights and additional hints, which is not available in vim/neovim. So both highlighting and inlay hints are not implemented. I'll follow up on this.